modules/list-providers-service/src/main/java/org/opencastproject/list/impl/EmptyResourceListQuery.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * Licensed to The Apereo Foundation under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ *
+ * The Apereo Foundation licenses this file to you under the Educational
+ * Community License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License
+ * at:
+ *
+ *   http://opensource.org/licenses/ecl2.txt
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ */
+
+package org.opencastproject.list.impl;
+
+import org.opencastproject.list.api.ResourceListFilter;
+import org.opencastproject.list.api.ResourceListQuery;
+import org.opencastproject.util.data.Option;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * ResourceListQuery that should result in an empty list after execution.
+ */
+public final class EmptyResourceListQuery implements ResourceListQuery {
+  private static final EmptyResourceListQuery instance = new EmptyResourceListQuery();
+
+  private EmptyResourceListQuery() {
+  }
+
+  public static ResourceListQuery get() {
+    return instance;
+  }
```

Comment:
Why are you implementing a Singleton?

Reply:
Do we need multiple instances of this class?

Reply:
Maybe not but it does not have to ensure that only one instance exists.